tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -71,6 +74,20 @@ public:
       virtual void Wait() = 0;
    };
 
+   /// A sealed page contains the bytes of a page as written to storage (packed & compressed).  It is used
+   /// as an input to UnsealPages() as well as to transfer pages between different storage media.
+   struct RSealedPage {
+      const void *fBuffer = nullptr;
```

Comment:
Is known whether this buffer is "owned" by the RSealedPage (as in "as the same lifetime as") or is it registered/owner by another entity?  Either way, adding a comment to indicate the answer would be useful.

Reply:
It's not owned, clarified in the class comment.